lrtable/src/lib/statetable.rs

```diff
@@ -85,7 +85,9 @@ pub struct StateTable<StorageT> {
     //   0  shift 1
     //   1  shift 0  reduce B
     // is represented as a hashtable {0: shift 1, 2: shift 0, 3: reduce 4}.
-    actions: HashMap<u32, Action<StorageT>, BuildHasherDefault<FnvHasher>>,
+    // Present as bits
+    // Action (4), Argument (max(pdix, stidx))
```

Comment:
This comment doesn't seem to match the `encode` function?

Reply:
Oh, yes, that is a left over and should be deleted.